source/guide_goaccess.rst

```diff
@@ -0,0 +1,199 @@
+.. highlight:: console
+
+.. author:: FM <git.fm@mmw9.de>
+
+########
+GoAccess
+########
+
+GoAccess_ is an open source realtime web log analyzer and viewer. It use the standard web server log file as source to build a readable graphical HTML output as web page (dashboard) or an overview in the shell. 
+
+This documentation describe the way to let create a static web page as overview, which will be updated every hour with a cron job. More details are on the `man page`_ and on GitHub_ available.
+
+License
+=======
+
+GoAccess is released under the `MIT License`_
+
+Installation
+============
+
+Step 1 - Download and Extract the Source Code
+---------------------------------------------
+
+Create a working directory.
+
+::
+
+ [isabell@stardust ~]$ mkdir ~/filebase/
```

Comment:
What's "filebase"? :sweat_smile: `~/goaccess` (or even `~/tmp/goaccess-x.y`, if we switch to tarballs) seems to be a good place to put the source. What do you think?